### PR TITLE
GH1602: Fix MSBuild file logger verbosity issue

### LINKED ISSUE #1602: MSBuildFileLogger Verbosity does not accept Verbosity.Verbose
```csharp
Task("Build")
    .IsDependentOn("Restore-NuGet-Packages")
    .Does(() =>
{
    MSBuild(solution, settings =>
        settings.SetConfiguration(configuration)
            .WithProperty("TreatWarningsAsErrors", "True")
            .SetVerbosity(Verbosity.Minimal)
            .AddFileLogger(new MSBuildFileLogger
            {
                Verbosity = Verbosity.Verbose
            }));
});
```

results in

```
========================================
Build
========================================
Executing task: Build
Microsoft (R) Build Engine version 15.1.1012.6693
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : Logger error MSB4103: "Verbose" is not a valid logger verbosity level.
An error occurred when executing task 'Build'.
Error: MSBuild: Process returned an error (exit code 1).
```

However, `Verbosity.Diagnostic` works great!

### PR DESCRIPTION
**Pull request by devlead**
* fixes #1602

### CODE REVIEW

File: src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs [hunk 1]
```diff
@@ -198,7 +198,7 @@ private static string GetPlatformName(PlatformTarget platform, bool isSolution)
             }
         }
 
-        private static string GetVerbosityName(Verbosity verbosity)
+        internal static string GetVerbosityName(Verbosity verbosity)
```
Comment (mholo65) [inline src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs]: Should this be moved elsewhere? Feels kind of hacky to call `MSBuildRunner.GetVerbosityName(...)`?

Comment (mholo65) [reply src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs]: Could it be an extension method for `Verbosity` instead?

Comment (devlead) [reply src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs]: Agreed, perhaps just an extension method on Verbosity?

Comment (devlead) [reply src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs]: Yep will fix that 😎

Comment (devlead) [reply src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs]: @mholo65 Addressed & rebased.

File: src/Cake.Common/Tools/MSBuild/MSBuildVerbosityExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    /// Contains functionality related to MSBuild verbosity.
+    /// </summary>
+    public static class MSBuildVerbosityExtensions
+    {
+        /// <summary>
+        /// Gets the MSBuild verbosity from <see cref="Verbosity"/>.
+        /// </summary>
+        /// <param name="verbosity">The verbosity.</param>
+        /// <returns>MSBuild verbosity string.</returns>
+        public static string GetMSBuildVerbosityName(this Verbosity verbosity)
```
Comment (mholo65) [inline src/Cake.Common/Tools/MSBuild/MSBuildVerbosityExtensions.cs]: Just one thought. This was `private` before, now public. And as a extension method in a namespace that is imported automatically with the alias. Intellisense will find this :smile:

Any issues with this?

Comment (devlead) [reply src/Cake.Common/Tools/MSBuild/MSBuildVerbosityExtensions.cs]: Don't think so, might even be useful 😎 but could of course internalize?

**Comment by mholo65** [comment]
@devlead your changes have been merged. Thank you for your contribution.

**Pull request merged**